utils/docker/base-rt/Dockerfile

```diff
@@ -30,7 +30,7 @@ ARG UBUNTU_VERSION
 ARG UBUNTU_NAME
 
 # config/build params (part 2)
-ARG SGX=2.12
+ARG SGX=2.16
```

Comment:
Is this version the most recent one we can get without "compatibility" issues / api changes?